### DISCUSSION TITLE
Is there a Linux tool similar to 'w' that shows users logged in via xrdp?

### DISCUSSION THREAD

**Discussion by bo-qeye** [neutrinolabs/xrdp#3215] [General]
If one runs the 'w' command on a server providing xrdp, it does not show any users logged in via XRDP. Is there a way to see this?

**Comment by matt335672** [comment]
Support was merged in #2745 but that isn't available in a release version yet - only `devel`.

**Reply by bo-qeye** [reply]
Thanks, this is actually super nifty! Looking forward to this getting merged/released.

**Reply by matt335672** [reply]
You can get some of it working by adding the right entries to /etc/pam.d/xrdp-sesman. See #1302 for more info.

**Comment by alexmyczko** [comment]
maybe this is useful? https://github.com/alexmyczko/autoexec.bat/blob/master/xrdpctl
